src/components/RecordsDetail/index.js

```diff
@@ -211,6 +211,10 @@ const RecordsDetail = props => {
                 params={props.params}
                 title='Description'
                 text={props.item.description} />
+              <PanelTextSection
+                params={props.params}
+                title='Description'
+                text={noteTextByType(props.item.notes, 'bioghist')} />
```

Comment:
```suggestion
              <PanelTextSection
                params={props.params}
                title='Biographical/Historical Notes'
                text={noteTextByType(props.item.notes, 'bioghist')} />
```

Reply:
Looking to avoid double description headings: ![image](https://user-images.githubusercontent.com/15254041/107698563-ba2d2080-6c82-11eb-8e93-02d5cc862c2e.png)

Reply:
🤦 Good call - got a little too trigger happy on the cut and paste there...